packages/typespec-ts/src/modular/buildCodeModel.ts

```diff
@@ -754,6 +755,7 @@ function emitBasicOperation(
   rlcModels: RLCModel,
   hierarchies: string[]
 ): HrlcOperation {
+  const getSdkType = useSdkTypes()!;
```

Comment:
Changes here are an example on how to use this new context.

/cc - @qiaozha, @MaryGao